content/en/docs/tasks/security/authz-tcp/index.md

```diff
@@ -29,23 +29,23 @@ the product page, you can see the following sections:
 When you refresh the page, the app shows different versions of reviews in the product page.
 The app presents the reviews in a round robin style: red stars, black stars, or no stars.
 
-## Installing and configuring a TCP service
+## Installing and configuring a TCP workload
 
-By default, the [Bookinfo](/docs/examples/bookinfo/) example application only includes HTTP services.
-To show how Istio handles the authorization of TCP services, we must update the application to use a
-TCP service. Follow this procedure to deploy the Bookinfo example app and update its `ratings` service
+By default, the [Bookinfo](/docs/examples/bookinfo/) example application only includes HTTP protocol.
+To show how Istio handles the authorization of TCP protocol, we must update the application to use a
+TCP protocol. Follow this procedure to deploy the Bookinfo example app and update its `ratings` workload
 to the `v2` version, which talks to a MongoDB backend using TCP.
 
-1. Install `v2` of the `ratings` service with service account `bookinfo-ratings-v2`:
+1. Install `v2` of the `ratings` workload with service account `bookinfo-ratings-v2`:
```

Comment:
You already described this information in the paragraph above. This is not a step, it is an introduction to a list of other steps, since achieving this task takes more than one command or there is more than one set of commands to do it.

Reply:
What is the suggestion here?

Reply:
I see you already split the explanation from the step, which was the original issue.
This step just needs some reordering:
```suggestion
1. Install `v2` of the `ratings` workload with the `bookinfo-ratings-v2` service account:
```